randovania/data/json_data/prime3.txt

```diff
@@ -3549,22 +4385,38 @@ Asset id: 16389095850931092901
 > First Floor; Heals? False
   > Dock to Processing Access
       Trivial
+  > Pickup (Missile Expansion)
+      After Metroid Processing Disable Barrier
   > Top Floor
-      Spider Ball
+      Any of the following:
+          After Metroid Processing Ball Lift
+          Spider Ball and Boost Ball
   > Basement
       X-Ray Visor
 
 > Basement; Heals? False
   > Pickup (Energy Cell)
       Trivial
+  > First Floor
+      Trivial
+  > Event - Ball Lift
+      Plasma Beam
+
+> Event - Metroid Barrier; Heals? False
+  * Event Metroid Processing Disable Barrier
+  > Basement
+      Trivial
+
+> Event - Ball Lift; Heals? False
+  * Event Metroid Processing Ball Lift
```

Comment:
Event for Ball Lift is missing a connection to a node. Connect it to Basement as Trivial

Reply:
Should already be there - might've made this comment before I made another push with minor cleanup. Feel free to resolve this after my next push to confirm this.

Reply:
Yeah this was different when I looked at it. It's resolved now.